ansible/roles/nginx/tasks/nginx_configs.yml

```diff
@@ -131,3 +156,41 @@
   notify: [ 'Test nginx and reload' ]
   when: (item.name|d() and item.state|d('present') != 'absent' and
          (item.delete is undefined or not item.delete|bool))
+
+- name: Create H5BP directory
+  file:
+    path: '/etc/nginx/conf.d/h5bp/'
+    state: directory
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+  when: (item.name|d() and item.state|d('present') != 'absent' and
+         (item.delete is undefined or not item.delete|bool) and
+         (nginx__h5bp_enabled|bool))
+
+- name: Create H5BP sub-directories
+  file:
+    path: '/etc/nginx/conf.d/h5bp/{{ item.name | dirname }}'
+    state: directory
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+  with_flattened:
+    - '{{ nginx__h5bp_files }}'
+  when: (item.name|d() and item.state|d('present') != 'absent' and
+         (item.delete is undefined or not item.delete|bool) and
+         (nginx__h5bp_enabled|bool))
+
+- name: Add HTML5 Boilerplate Files
+  get_url:
+    url: https://raw.githubusercontent.com/h5bp/server-configs-nginx/{{ nginx__h5bp_commit }}/h5bp/{{ item.name }}
+    dest: '/etc/nginx/conf.d/h5bp/{{ item.name }}'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
+  with_flattened:
+    - '{{ nginx__h5bp_files }}'
+  notify: [ 'Test nginx and reload' ]
+  when: (item.name|d() and item.state|d('present') != 'absent' and
+         (item.delete is undefined or not item.delete|bool) and
+         (nginx__h5bp_enabled|bool))
```

Comment:
I would prefer when the https://github.com/h5bp/server-configs-nginx gets cloned below `/usr/local/src` on the remote system and then the files are copied in place. Search for `/usr/local/src` in the DebOps repo for examples.

See also: https://docs.debops.org/en/master/dep/dep-0003.html